catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/model/pojo/FieldFilter.java

```diff
@@ -35,10 +38,26 @@
   @SuppressWarnings("squid:S1068" /* Needed for serialization */)
   private final Set<String> descriptors;
 
-  public FieldFilter(Metacard metacard, Set<String> descriptors) {
+  @JsonProperty("creator")
+  private String creator;
+
+  @JsonProperty("accessIndividuals")
```

Comment:
❓ Do we actually need these `@JsonProperty` annotations? The fact that they weren't there already (and aren't in the `CommonTemplate` class this extends from) makes me think we're likely going about this a different way anyway

Reply:
The annotations are added to be explicit with the naming convention. For example, `root` differs from the desired JSON variable of `filterTemplate`. I was under the assumption the annotations were required if getters didn't exist in the class. I will confirm if that's true or not.